packages/handlers/grpc/src/index.ts

```diff
@@ -84,21 +86,36 @@ export default class GrpcHandler implements MeshHandler {
     });
 
     const root = new Root();
-    let fileName = this.config.protoFilePath;
-    let options: LoadOptions = {};
-    if (typeof this.config.protoFilePath === 'object' && this.config.protoFilePath.file) {
-      fileName = this.config.protoFilePath.file;
-      options = this.config.protoFilePath.load;
-      if (options.includeDirs) {
-        if (!Array.isArray(options.includeDirs)) {
-          return Promise.reject(new Error('The includeDirs option must be an array'));
+    let packageDefinition: PackageDefinition;
+    if (this.config.useReflection) {
+      const grpcReflectionServer = this.config.endpoint;
+      const reflectionClient = new grpcReflection.Client(grpcReflectionServer, creds);
+      const services = (await reflectionClient.listServices()) as string[];
+      const serviceRoots = await Promise.all(
+        services.map((service: string) => reflectionClient.fileContainingSymbol(service))
+      );
+      serviceRoots.forEach((serviceRoot: Root) => {
+        serviceRoot.name = this.config.serviceName || '';
```

Comment:
By default, the first `nested` object will have a name of `""`. We add the `config.serviceName` to properly nest the objects for naming.